documentation/frontend/featureFlags.md

```diff
@@ -0,0 +1,162 @@
+# Feature flags
+
+This feature flags implementation stores feature flags settings on user cookies. The intent is for these feature flags
+to be user configurable, so that 1. we can gate incomplete features to make it easier to coordinate development work,
+and 2. links can be sent to individuals with customized query parameters to customize the user's experience.
+
+For codebase/deployment level feature flagging that isn't meant for users to be able to configure, those continue to be
+implemented using `process.env`.
+
+## Feature flags in use
+
+As of February 2024, the following feature flags are set in the project:
+1. hideSearch - 
+  * Defaults to true
+  * This is for hiding the search page as it is being developed and user tested
+  * This should be removed when the search page goes live, before May 2024
```

Comment:
nit: maybe have a specific js file that stores and documents the flags, and then have `FeatureFlagManager` pull them in

// featureFlags.js
const featureFlags = {
   // This is for hiding the search page (ticket #, etc...)
   // This should be removed when the search page goes live, before May 2024
   hideSearch : true,
}

Just a nit though